crates/turbopack-ecmascript/src/analyzer/mod.rs

```diff
@@ -1371,21 +1938,102 @@ impl JsValue {
         }
     }
 
+    /// Calls a function for only early or lazy children. Allows mutating the
+    /// node. Updates the total nodes count after mutation.
+    pub fn for_each_early_children_mut(
```

Comment:
This should be two different functions: one for `early: true` and one for `early: false`.